pkg/scheduler/scheduler_test.go

```diff
@@ -635,7 +640,7 @@ func TestSchedule(t *testing.T) {
 			if err != nil {
 				t.Fatalf("Failed setting up watch: %v", err)
 			}
-			scheduler := New(qManager, capCache, cl)
+			scheduler := New(qManager, capCache, cl, recorder)
```

Comment:
We can unit tests events too

Reply:
doesn't seem necessary, feels like it will clutter tests.